qi-doc/scribblings/field-guide.scrbl

```diff
@@ -65,9 +65,9 @@ Side effects are a natural fit for debugging functional code in general, as the
 
 @defmodule[qi/probe]
 
-Qi includes a "circuit tester" style debugger, which you can use to check the values at arbitrary points in the flow. It can be used even if the flow is raising an error – the tester can help you find the error. It offers similar functionality to @seclink["top" #:indirect? #t #:doc '(lib "debug/scribblings/debug.scrbl")]{debug} but is specialized for functional debugging and Qi flows.
+Qi includes a "circuit tester" style debugger, which you can use to check the values at arbitrary points in the @tech{flow}. It can be used even if the @tech{flow} is raising an error – the tester can help you find the error. It offers similar functionality to @seclink["top" #:indirect? #t #:doc '(lib "debug/scribblings/debug.scrbl")]{debug} but is specialized for functional debugging and Qi flows.
 
-To use it, first wrap the entire expression @emph{invoking} the flow with a @racket[probe] form. Then, you can place a literal @racket[readout] anywhere within the flow definition to cause the entire expression to evaluate to the values flowing at that point. This works even if your flow is defined elsewhere (even in another file) and only @emph{used} at the invocation site by name.
+To use it, first wrap the entire expression @emph{invoking} the @tech{flow} with a @racket[probe] form. Then, you can place a literal @racket[readout] anywhere within the @tech{flow} definition to cause the entire expression to evaluate to the values flowing at that point. This works even if your @tech{flow} is defined elsewhere (even in another file) and only @emph{used} at the invocation site by name.
```

Comment:
For comparison, I looked at the Wikipedia page for [Division](https://en.wikipedia.org/wiki/Division_(mathematics)), and searched for occurrences of "quotient." It's not super consistent but it appears they hyperlink it more like once per section there, so maybe that would be sufficient?

Reply:
Another comparison, "exception handler" on this page about [Exceptions in the Racket docs](https://docs.racket-lang.org/reference/exns.html). It's about once per section there or even less (but again not super consistent).

Reply:
Does `set linebreak` help? I personally use this config:

```
  set linebreak           " (soft-) wrap lines on words
  set breakindent         " wrap lines taking indentation into account
  set showbreak=..        " wrap lines taking indentation into account
```

and make heavy use of `gj` and `gk`. Otherwise, it seems to get annoying to constantly `gq` paragraphs. How do you get around this?

Reply:
I do actually use those settings. I run Vim in a full-screen terminal: on my laptop I typically have a fair amount of columns available (on my usual monitor I have almost 300 at my current font-size). So I still have to scan long lines unless I do weird things like split a window just to control the width.

So… actually, I do just `gqip` a lot. Examples: https://github.com/benknoble/frosthaven-manager/blob/main/scribblings/how-to-play.scrbl, https://github.com/benknoble/funarch-2023-gui-easy/blob/main/report.scrbl

At any rate, I probably won't get around to cleaning this up myself.

Reply:
Ok, no worries. I tend to use widely varying (and typically high) zoom levels so it generally looks like a nicely justified paragraph. But it also means that `gq`'d paragraphs get weird, since they soft-wrap at the window column but then hard-wrap at the real end of line. This can either make them underflow or overflow depending on the zoom level. But if explicitly justified text is more widely preferred, I'm happy to accommodate.

At first I thought a "soft wrap at column" feature could solve this issue to our mutual satisfaction, but when I googled that, I found myself [continuing this conversation with you](https://vi.stackexchange.com/questions/43028/how-to-soft-wrap-text-at-column-number-lower-than-window-width), and sounds like there isn't a good way to do that! Kind of surprising, but I guess differently sized monitors and zoom levels is a relatively modern phenomenon.

Reply:
Hah! I havent seen the plugin mentioned in that Q&A, but it's an interesting (ab)use of virtual text. 

Yes, you're certainly correct that hard-wrapped lines do poorly on narrow screens (my phone's email client provides evidence of that anytime I look at hard-wrapped email).